Gemfile

```diff
@@ -94,6 +94,7 @@ end
 
 group :development do
   gem 'better_errors'
+  gem 'binding_of_caller'
```

Comment:
Remove this is for debug only, not production.

Reply:
This will only install for development don't worry!